signer/src/lib.rs

```diff
@@ -320,7 +320,7 @@ fn verify_secp256k1_signature(
     let blob_to_sign = Message::parse_slice(&message_digest)?;
 
     let public_key = recover(&blob_to_sign, &signature_rs, &recovery_id)?;
-    let mut from = Address::new_secp256k1(&public_key.serialize_compressed().to_vec());
+    let mut from = Address::new_secp256k1(&public_key.serialize().to_vec());
```

Comment:
this is also a breaking change... mmm... are we sure about this? There were some problems in the past because of this...

Reply:
yes, now I remember. This is definitely a bug.
I had trouble in the past, and requested the spec team to make this explicit

https://github.com/filecoin-project/specs/pull/872#issue-369581772

Reply:
Yeah that was weird. That's why also I had to change tests were I was verifying signatures. They should check length.

It was actually the only place where it was using compress key so no big deal.

Reply:
Austin did it already :)
https://github.com/ChainSafe/forest/pull/438